test/src/main/java/org/apache/accumulo/harness/TestingKdc.java

```diff
@@ -52,48 +54,54 @@ public class TestingKdc {
   public final String ORG_NAME = "EXAMPLE", ORG_DOMAIN = "COM";
 
   private String hostname;
-  private File keytabDir;
+  private Path keytabDir;
   private boolean started = false;
 
   public TestingKdc() throws Exception {
     this(computeKdcDir(), computeKeytabDir(), MAX_TICKET_LIFETIME_MILLIS);
   }
 
-  public static File computeKdcDir() {
+  public static Path computeKdcDir() throws IOException {
     Path userDir = Path.of(System.getProperty("user.dir"));
-    File targetDir = userDir.resolve("target").toFile();
-    if (!targetDir.exists()) {
-      assertTrue(targetDir.mkdirs());
+    Path targetDir = userDir.resolve("target");
+    if (!Files.exists(targetDir)) {
+      Files.createDirectories(targetDir);
     }
-    assertTrue(targetDir.isDirectory(), "Could not find Maven target directory: " + targetDir);
+    assertTrue(Files.isDirectory(targetDir), "Could not find Maven target directory: " + targetDir);
 
     // Create the directories: target/kerberos/minikdc
-    File kdcDir = targetDir.toPath().resolve("kerberos").resolve("minikdc").toFile();
+    Path kdcDir = targetDir.resolve("kerberos").resolve("minikdc");
 
-    assertTrue(kdcDir.mkdirs() || kdcDir.isDirectory());
+    if (!Files.isDirectory(kdcDir)) {
+      Files.createDirectories(kdcDir);
+    }
 
     return kdcDir;
   }
 
-  public static File computeKeytabDir() {
+  public static Path computeKeytabDir() throws IOException {
     Path userDir = Path.of(System.getProperty("user.dir"));
-    File targetDir = userDir.resolve("target").toFile();
-    assertTrue(targetDir.exists() && targetDir.isDirectory(),
-        "Could not find Maven target directory: " + targetDir);
+    Path targetDir = userDir.resolve("target");
+    if (!Files.exists(targetDir)) {
+      Files.createDirectories(targetDir);
+    }
+    assertTrue(Files.isDirectory(targetDir), "Could not find Maven target directory: " + targetDir);
```

Comment:
This kind of thing is done a lot. I wonder if you can move this to a method in some file utility class